package.json

```diff
@@ -1,4 +1,8 @@
 {
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node dist/server.js"
```

Comment:
Will starting the server this way work in production?

Reply:
asking because we may need to have 2 separate scripts, one for dev and one for prod, since prod contains only whats inside the dist folder.

Reply:
There is a script for development pushed in #7 that watches for changed files and reruns the server.